content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst

```diff
@@ -55,77 +55,61 @@ A :guilabel:`Receipt` smart button will appear in the top right, and the receipt
 with the purchase order. Clicking on the :guilabel:`Receipt` smart button will show the receipt
 order.
 
-.. image:: receipts_three_steps/three-step-purchase-receipt.png
+.. image:: receipts_three_steps/receipts-three-steps-smart-button.png
```

Comment:
Consider reverting here, this revision is a bit too zoomed in so we're missing navigational context from the surrounding UI elements.

Reply:
Screenshot replaced with new zoomed-out one that shows navigational UI and highlights key areas. Suggestion implemented ✅